container_templates/matlab/container_example/Dockerfile

```
@@ -0,0 +1,49 @@
+# Modified from https://github.com/mathworks-ref-arch/matlab-dockerfile.
+
+# To specify which MATLAB release to install in the container, edit the value of the MATLAB_RELEASE argument.
+ARG MATLAB_RELEASE=r2021b
+
+# When you start the build stage, this Dockerfile by default uses the Ubuntu-based matlab-deps image.
+# To check the available matlab-deps images, see: https://hub.docker.com/r/mathworks/matlab-deps
+FROM mathworks/matlab-deps:${MATLAB_RELEASE}
```

```suggestion
FROM mathworks/matlab:${MATLAB_RELEASE}
```

I think that Matlab now provides pre-built containers so you don't even need to run the mpm stuff below to build your container anymore... https://hub.docker.com/r/mathworks/matlab

So I'm wondering if you can get rid of all the installation work below and then leave in your ARG and COPY examples below.